src/hesiod.c

```diff
@@ -119,6 +121,23 @@ hsk_hesiod_resolve(hsk_dns_req_t *req, hsk_ns_t *ns) {
   msg->flags |= HSK_DNS_AA;
   hsk_dns_rrs_t *an = &msg->an;
 
+
+  // POOL
+  //  SIZE
+  if (hsk_dns_is_subdomain(req->name, "size.pool.hnsd.")) {
+    char size[65];
+    sprintf(size, "%d", ns->pool->size);
+    if (!hsk_hesiod_txt_push("size.pool.hnsd.", size, an))
+      goto fail;
+  }
+
+  //  PEERS
```

Comment:
Since `name_root` is under `tip.chain.hnsd`, it can probably be moved down under the CHAIN/TIP comment?


```suggestion
  //  NAME ROOT
```